### DISCUSSION TITLE
sjtubeamermin v1.0

### DISCUSSION THREAD

**Discussion by LogCreative** [LogCreative/SJTUBeamermin#16] [General]
> Pre-Release for CTAN

`sjtubeamermin` 来到了发布正式版前的最后版本。这个版本：
- 更新了强调命令 `\emph{}` 和 `\alert{}`。
- 添加了高亮命令 `\highlight{}` 和 `\paragraph{}`。
- 代码用 `l3build` 重构，开发文档重新编写。

将在不久后上传 CTAN，在此之前可能仍然会有变动。

<hr /><em>This discussion was created from the release <a href='https://github.com/LogCreative/SJTUBeamermin/releases/tag/1.0'>sjtubeamermin v1.0</a>.</em>

**Comment by AnkorTn** [comment]
It's very useful!!! Thank you very much!!!!
